src/ripple/nodestore/impl/DecodedBlob.cpp

```diff
@@ -21,6 +21,7 @@
 #include <ripple/nodestore/impl/DecodedBlob.h>
 #include <algorithm>
 #include <cassert>
+#include <iostream>
```

Comment:
I don't think this `#include` is necessary.

Reply:
removed